src/app/features/login/components/login/login.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="login__container">
+  <h2 class="login__header">Login</h2>
+  <form (submit)="onSubmit(username, password)" class="login__form">
```

Comment:
Get this `username, password` inside of onSubmit function, don't pass it as a param